Jellyfin.Server/Program.cs

```diff
@@ -273,9 +281,8 @@ private static IWebHostBuilder CreateWebHostBuilder(ApplicationHost appHost, ISe
                         }
                     }
                 })
-                .ConfigureAppConfiguration(config => config.ConfigureAppConfiguration(appPaths))
+                .ConfigureAppConfiguration(config => config.ConfigureAppConfiguration(commandLineOpts, appPaths, startupConfig))
                 .UseSerilog()
-                .UseContentRoot(appHost.ContentRoot)
```

Comment:
From what I can tell this call to `UseContentRoot` is incorrect and so I have removed it. The web host content root property is documented here: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/?view=aspnetcore-3.1&tabs=windows#content-root